Makefile

```
@@ -226,9 +230,13 @@ install-crds: manifests
 install-resources:
 	@echo creating namespaces
 	kubectl create ns policy-propagator-test
+	kubectl create ns $(KIND_NAMESPACE)
 	kubectl create ns managed1
 	kubectl create ns managed2
 	kubectl create ns managed3
+	@echo deploying roles and service account
+	kubectl apply -k deploy/rbac
```

I think this needs a namespace for the leader election role?

```suggestion
	kubectl apply -k deploy/rbac -n $(KIND_NAMESPACE)
```